docs/docs/rust.md

```diff
@@ -0,0 +1,3 @@
+# Rust API documentation
```

Comment:
This should be called "DBSP Low-level API", and the file shouldn't be `rust.md` as an end-user I don't really care it's in rust